package/MDAnalysis/lib/distances.py

```diff
@@ -208,6 +208,23 @@ def _check_lengths_match(*arrays):
         raise ValueError("Input arrays must all be same shape"
                          "Got {0}".format([a.shape for a in arrays]))
 
+
+def distance(a, b, box=None):
+    """calc distances between two points. Uses algorithm from Tuckerman
```

Comment:
Doc string should state that this takes any PBC into account. I would also reference Tuckerman properly (and possibly even write out the math because we would want to use it as a test case for correctness and in this case, one cannot be explicit enough.)